helm/prometheus-rules/templates/alerting-rules/kyverno.all.rules.yml

```diff
@@ -42,6 +42,21 @@ spec:
         severity: page
         team: shield
         topic: kyverno
+    - alert: KyvernoUpdateRequestsCountTooHigh
+      annotations:
+        description: "{{`Kyverno {{ $labels.kind }} are too high. This is an indicator that Kyverno\'s background controller may not be able to create some resources.`}}"
```

Comment:
Why did you add a \ here ? Otherwise LGTM

Reply:
Not sure, I copied it from another rule, will try without it